src/domain/space/space/space.service.authorization.ts

```
@@ -463,61 +428,47 @@ export class SpaceAuthorizationService {
+      [];
+    switch (spaceSettings.privacy.mode) {
+      // Also for PUBLIC spaces cascade the read on About to avoid having privilege rules everywhere
+      case SpacePrivacyMode.PUBLIC:
+      case SpacePrivacyMode.PRIVATE:
+        const credentialRuleReadOnAbout =
+          this.authorizationPolicyService.createCredentialRule(
+            [AuthorizationPrivilege.READ],
+            credentialCriteriasWithAccess,
+            'Read access to About'
+          );
+        credentialRuleReadOnAbout.cascade = true;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)